sass/components/_popover.scss

```diff
@@ -233,6 +233,7 @@
 }
 
 .user-popover__role {
+    background: rgba(var(--center-channel-color), 0.3);
```

Comment:
You should use the `center-channel-color-rgb` variable.

Reply:
Thanks for review!